docker/Dockerfile

```diff
@@ -15,15 +15,28 @@ SHELL ["/bin/sh", "-c"]
 RUN groupadd docker && \
     usermod -aG docker viadot
 
-# Release File Error
-# https://stackoverflow.com/questions/63526272/release-file-is-not-valid-yet-docker
-RUN echo "Acquire::Check-Valid-Until \"false\";\nAcquire::Check-Date \"false\";" | cat > /etc/apt/apt.conf.d/10no--check-valid-until
```

Comment:
This was needed in the past to be able to reliably build the image on WSL, since WSL hardware clock would sometimes get out of sync and come random issues such as this one.